TESTS/mbed_hal/flash/functional_tests/main.cpp

```diff
@@ -52,17 +52,57 @@ static void erase_range(flash_t *flash, uint32_t addr, uint32_t size)
         size = size > sector_size ? size - sector_size : 0;
     }
 }
+#ifdef __CC_ARM
+MBED_NOINLINE
+__asm static void delay_loop(uint32_t count)
+{
+1
```

Comment:
I got a question regarding this 1, is it required? I came across this code today, and wondering if this is line is needed?